### DISCUSSION TITLE
How to pass options for WEBP format the same way as using cwebp tool

### DISCUSSION THREAD

**Discussion by Myslik** [dlemstra/Magick.NET#861] [Help]
Hello,
I am trying to use `-sharp_yuv` option using Magick.NET using following code.

```csharp
using (var targetImage = new MagickImage("wizard:"))
{
    targetImage.Quality = 90;
    targetImage.Settings.SetDefine(MagickFormat.WebP, "use_sharp_yuv", true);
    targetImage.Write(targetPath);
}
```

Am I doing this right? It seems that image is different than what `cwebp` generates.

**Comment by dlemstra** [comment]
You should be able to use `webp:use-sharp-yuv` for this. I should probably add a `WebPWriteDefines` class to make this easier.
